examples/inbound-deliveries-to-sap/types.bal

```diff
@@ -0,0 +1,25 @@
+type S4HANAClientConfig record {|
```

Comment:
License

Reply:
Updated in [Update reviewed changes](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/16/commits/8650b99db9b8e7a6078a7120025cc0bf81979c17)